common/lib/xmodule/xmodule/util/xmodule_django.py

```diff
@@ -5,8 +5,10 @@
 """
 
 # NOTE: we are importing this method so that any module that imports us has access to get_current_request
-from crum import get_current_request
```

Comment:
isort didn't move the comment that refers to this line, could you update this to keep them together?